nav2_system_tests/src/waypoint_follower/test_case_py.launch

```diff
@@ -53,11 +55,17 @@ def generate_launch_description():
         SetEnvironmentVariable('RCUTILS_LOGGING_BUFFERED_STREAM', '1'),
         SetEnvironmentVariable('RCUTILS_LOGGING_USE_STDOUT', '1'),
 
+        DeclareLaunchArgument(
+            'world',
```

Comment:
Where is `world` used? The below used to use it for launching gazebo, but not launches `gzserver.launch.py` and I don't see where `world` is given to it

Reply:
As far as I understood, in that way `world` is set globally like an environment variable, then it's used from `gzserver.launch.py`. 

Also in `small_warehouse.launch.py` `world` is set this way.

Reply:
is there a way to make that more explicit, like giving it the `world` as an argument, versus implicitly setting it through that interface?

Reply:
I haven't found a way to pass `world` file as an argument to any of Gazebo `.launch.py`. The only way is from command line launch `ros2 launch gazebo_ros gazebo.launch.py world:=<world file>`, which is basically the same of setting `world` as `LaunchConfiguration` I think.

Reply:
https://github.com/ros-simulation/gazebo_ros_pkgs/blob/galactic/gazebo_ros/launch/gzserver.launch.py#L84 you could pass it as a launch argument as we would any other launch argument. Example: https://github.com/ros-planning/navigation2/blob/main/nav2_bringup/launch/tb3_simulation_launch.py#L197

Reply:
Fixed.